source/extensions/access_loggers/grpc/grpc_access_log_utils.cc

```diff
@@ -239,9 +239,9 @@ void Utility::extractCommonAccessLogProperties(
   }
 
   for (const auto& key : config.filter_state_objects_to_log()) {
-    if (stream_info.filterState().hasDataWithName(key)) {
+    if (stream_info.filterState()->hasDataWithName(key)) {
```

Comment:
It appears https://github.com/envoyproxy/envoy/pull/9845 added this ambiguity causing the windows CI compilation failures, not yet quite sure why MSVC is able to resolve it here: https://github.com/envoyproxy/envoy/pull/9845/files#diff-a5570d8b9053a47a1547728cb6c90989 and not this case